src/sidebar/components/NotebookView.tsx

```diff
@@ -133,8 +134,14 @@ function NotebookView({ loadAnnotationsService, streamer }: NotebookViewProps) {
           {groupName}
         </h1>
       </header>
-      <div className="absolute w-full z-5 left-0 top-7">
-        <div className="container flex flex-row-reverse">
+      <div className="absolute w-full z-5 left-0 lg:top-8 top-5">
+        <div
+          className={classnames(
+            'container flex flex-row-reverse relative',
+            // Compensate for container's right padding
```

Comment:
Can you add a comment here noting where this right padding is defined. The fact that `container` here refers to a Tailwind container, which in turn has styling customizations in the Tailwind config, was not obvious for me.